app/components/header.jsx

```diff
@@ -8,21 +8,21 @@ import moon from "../images/moon.png"
 
 export default function Header({ isDarkMode, setIsDarkMode }) {
     return (
-        <div className="flex items-center justify-between">
+        <div className="flex items-center justify-between w-full absolute">
             <div className="font-bold text-lg mt-4">
-                <Link className="text-2xl ml-6 mr-10 text-white hover:underline" href="/">Shop</Link>
-                <Link className="text-2xl mr-10 text-white hover:underline" href="https://acm.uic.edu/about">About</Link>
-                <Link className="text-2xl text-white hover:underline w" href='https://acm.uic.edu/about'>Contact</Link>
+                <Link href="/" className="text-2xl ml-6 mr-10 text-white hover:underline">Shop</Link>
+                <Link className="text-2xl mr-10 text-white hover:underline" href="https://acm.uic.edu/about" target="_blank">About</Link>
+                <Link className="text-2xl text-white hover:underline w" href='/Contributors'>Contact</Link>
```

Comment:
This link should still be to the ACM contact page as it was before. The website contributor link will be in the footer.